tests/networking_test.go

```diff
@@ -0,0 +1,81 @@
+package tests_test
+
+import (
+	v2vv1alpha1 "github.com/kubevirt/vm-import-operator/pkg/apis/v2v/v1alpha1"
+	"github.com/kubevirt/vm-import-operator/tests/framework"
+	. "github.com/kubevirt/vm-import-operator/tests/matchers"
+	vms "github.com/kubevirt/vm-import-operator/tests/ovirt-vms"
+	"github.com/kubevirt/vm-import-operator/tests/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "kubevirt.io/client-go/api/v1"
+)
+
+var _ = Describe("Import of VM ", func() {
+	var (
+		f           = framework.NewFrameworkOrDie("networking")
+		secret      corev1.Secret
+		namespace   string
+		networkName string
+	)
+
+	BeforeSuite(func() {
+		err := f.ConfigureNodeNetwork()
+		if err != nil {
+			Fail(err.Error())
+		}
+	})
+
+	BeforeEach(func() {
+		namespace = f.Namespace.Name
+		s, err := f.CreateOvirtSecretFromBlueprint()
+		if err != nil {
+			Fail("Cannot create secret: " + err.Error())
+		}
+		secret = s
+		nad, err := f.CreateLinuxBridgeNetworkAttachmentDefinition()
+		if err != nil {
+			Fail(err.Error())
+		}
+		networkName = nad.Name
+	})
+
+	Context("with multus network", func() {
+		It("should create running VM", func() {
+			vmi := utils.VirtualMachineImportCr(vms.BasicNetworkVmID, namespace, secret.Name, f.NsPrefix, true)
+			vmi.Spec.Source.Ovirt.Mappings = &v2vv1alpha1.OvirtMappings{
+				NetworkMappings: &[]v2vv1alpha1.ResourceMappingItem{
+					{Source: v2vv1alpha1.Source{ID: &vms.BasicNetworkID}, Type: &multusType, Target: v2vv1alpha1.ObjectIdentifier{
+						Name:      networkName,
+						Namespace: &f.Namespace.Name,
+					}},
+				},
+			}
+			created, err := f.VMImportClient.V2vV1alpha1().VirtualMachineImports(namespace).Create(&vmi)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(created).To(BeSuccessful(f))
+
+			retrieved, _ := f.VMImportClient.V2vV1alpha1().VirtualMachineImports(namespace).Get(created.Name, metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
+			vmBlueprint := v1.VirtualMachine{ObjectMeta: metav1.ObjectMeta{Name: retrieved.Status.TargetVMName, Namespace: namespace}}
+			Expect(vmBlueprint).To(BeRunning(f))
+
+			By("Having correct network configuration")
+			vm, _ := f.KubeVirtClient.VirtualMachine(namespace).Get(vmBlueprint.Name, &metav1.GetOptions{})
```

Comment:
since the VM is running, why not to examine the VM-Instance ? the status object of it should be updated with data relevant to new interface IIRC